action.php

```diff
@@ -629,6 +629,9 @@ protected function load_template() {
             '@TPLBASE@' => DOKU_BASE . 'lib/plugins/dw2pdf/tpl/' . $this->tpl . '/',
             '@TPLINC@'  => DOKU_INC . 'lib/plugins/dw2pdf/tpl/' . $this->tpl . '/'
         );
+		
+		global $INFO;
+		$replace['@AUTHOR@'] = $INFO['meta']['contributor'][$INFO['user']];
```

Comment:
Indentation is wrong.  Also, why isn't this part of the array setup above?

Reply:
This is true. I will correct the indentation and add this to the array above.